src/app/services/mail/mail.service.ts

```diff
@@ -572,6 +575,42 @@ export class MailService {
       }),
     )
   }
+
+  /**
+   * Sends a warning email to the admin and collaborators of the form when their current verified sms counts hits a limit
+   * @param form The form whose admin and collaborators will be issued a warning
+   * @param remainingSmsCounts The current total responses to the form
```

Comment:
The current total responses isn't the remaining sms count right?

Reply:
changed teh api (and updated the comment @__@)